packages/fxa-payments-server/Dockerfile

```diff
@@ -1,27 +1,59 @@
-FROM node:12-stretch AS node-builder
+FROM node:12-stretch-slim AS node-builder
 RUN groupadd --gid 10001 app  && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app
 RUN mkdir /fxa-content-server && chown -R app:app /fxa-content-server
+RUN apt-get -y update && apt-get -y install git && apt-get clean && rm -rf /var/lib/apt/lists
 USER app
 WORKDIR /app
 COPY ["fxa-payments-server/package*.json", "./"]
 RUN npm ci
 COPY ["fxa-payments-server/.storybook", ".storybook/"]
 COPY ["fxa-payments-server/public", "public/"]
 COPY ["fxa-payments-server/src", "src/"]
+
+#COPY --chown=app:app ["fxa-shared", "../fxa-shared/"]
+#WORKDIR /fxa-shared
+#USER root
+#RUN chown -R app:app /fxa-shared
+#USER app
+#RUN npm ci
```

Comment:
Delete these line if they aren't needed no mo'.

Reply:
Cool, I'm going to add a bug number too, as it's a fairly hacky looking  Dockerfile